packages/ember-testing/lib/helpers.js

```diff
@@ -268,18 +268,19 @@ asyncHelper('visit', visit);
 
 /**
   Clicks an element and triggers any actions triggered by the element's `click`
-  event.
+  event. Accepts `context` as an optional second argument.
 
   Example:
 
   ```javascript
-  click('.some-jQuery-selector').then(function() {
+  click('.some-jQuery-selector', 'body').then(function() {
     // assert something
   });
   ```
 
   @method click
   @param {String} selector jQuery selector for finding element on the DOM
+  @param {String} context for finding element on the DOM
```

Comment:
Since `context` is an optional parameter, this should be `[context]` to indicate an optional parameter.